### PR TITLE
Add tests for 1.4

### PR DESCRIPTION
**Pull request by seancribbs**

### CODE REVIEW

**Comment by seancribbs** [comment]
I have not seen any new riak_test come in for 1.4, we should go ahead and merge this and deploy.

File: db/schema.rb [hunk 1]
```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended to check this file into your version control system.
```
Comment (joedevivo) [inline db/schema.rb:12]: Good job!

**Comment by joedevivo** [comment]
+1

Looks great, makes me wonder how we ever lived without it!

File: db/schema.rb (hunk 1 above)
Comment (cmeiklejohn) [inline db/schema.rb:12]: :+1:

**Pull request merged**